src/core/main.qml

```diff
@@ -129,13 +129,14 @@ Window {
     GCAudio {
         id: backgroundMusic
         muted: !ApplicationSettings.isBackgroundMusicEnabled
+        volume: 0.53
```

Comment:
why not 0.5 directly?
can you make a readonly default value and assign it to the value instead of putting it here and below?

Then, it should be good to go, you can directly commit on the branch.
Thank you

Johnny

ps: as a more long time contributor (and you already have an account) please prefer differential for patches :)

Reply:
sure, I'll prefer the differential for patches from next tasks.